routers/web/repo/projects.go

```diff
@@ -137,6 +137,10 @@ func RenderNewProject(ctx *context.Context) {
 	ctx.Data["CardTypes"] = project_model.GetCardConfig()
 	ctx.Data["CanWriteProjects"] = ctx.Repo.Permission.CanWrite(unit.TypeProjects)
 	ctx.Data["CancelLink"] = ctx.Repo.Repository.Link() + "/projects"
+	ctx.Data["ProjectMarkdownPreviewURL"] = fmt.Sprintf("%s/markup", ctx.Repo.Repository.Link())
+	ctx.Data["ProjectMarkdownPreviewContext"] = ctx.Repo.Repository.Link()
+	ctx.Data["ProjectMarkdownPreviewMode"] = "comment"
```

Comment:
Here and other places: the mode shouldn't be "comment" because the description is rendered in original markdown format